easybuild/easyconfigs/g/GKlib-METIS/GKlib-METIS-5.1.1-GCCcore-13.3.0.eb

```
@@ -0,0 +1,39 @@
+# Author: J. Sassmannshausen (Imperial College Londoni/UK)
+
+easyblock = 'CMakeMake'
+
+name = 'GKlib-METIS'
+version = '5.1.1'
+
+homepage = 'https://github.com/KarypisLab/GKlib'
+description = """A library of various helper routines and frameworks used by
+many of the lab's software"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+sources = [
+    {
+        'source_urls': ['https://github.com/KarypisLab/GKlib/archive'],
+        'download_filename': 'METIS-v%(version)s-DistDGL-0.5.tar.gz',
+        'filename': '%(name)s-v%(version)s-DistDGL-0.5.tar.gz',
+    }
+]
+checksums = ['52aa0d383d42360f4faa0ae9537ba2ca348eeab4db5f2dfd6343192d0ff4b833']
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+    ('binutils', '2.42'),
+]
+
+dependencies = [
+    ('Python', '3.12.3'),
+]
+
```

re-instate configopts to build the shared library as well
```suggestion

configopts = ['-DBUILD_SHARED_LIBS=ON ', '-DBUILD_SHARED_LIBS=OFF ']

```